main.go

```diff
@@ -62,14 +60,35 @@ func main() {
 	}
 
 	// Handle flags
-	flag.StringVar(&bot.configFile, "config", defaultPath, "-config sets filepath to load configuration")
-	flag.StringVar(&bot.dataDir, "datadir", common.GetDefaultDataDir(runtime.GOOS), "default data directory for GoCryptoTrader files")
-	dryrun := flag.Bool("dryrun", false, "-dryrun does not save config.json file")
-	version := flag.Bool("version", false, "-version retrieves current GoCryptoTrader version")
-	verbosity := flag.Bool("verbose", false, "-verbose increases verbosity for GoCryptoTrader")
+	flag.StringVar(&bot.configFile,
+		"config",
+		defaultPath,
+		"-config sets filepath to load configuration")
```

Comment:
You can omit the -config in the flag description and below ones since -help will output it anyway